run.py

```diff
@@ -31,6 +31,13 @@
     default=None,
     help="Path to training data.",
 )
+@click.option(
+    "--state",
+    "-st",
+    type=str,
+    default=None,
+    help="The saved model state to be loaded for evaluation/resume training.",
```

Comment:
looks like the implementation in this PR is only for the evaluation side, right?

Reply:
Yes. I am adding the resuming today.